spec/index.bs

```diff
@@ -2400,6 +2401,8 @@ A {{GPUBindGroupLayout}} object has the following internal slots:
         <dfn>sampled texture validation</dfn>: There must be {{GPULimits/maxSampledTexturesPerShaderStage|GPULimits.maxSampledTexturesPerShaderStage}} or
             fewer |bindingDescriptor|s of type {{GPUBindingType/sampled-texture}} visible on each shader stage in |descriptor|.
 
+        <dfn>multisample texture view dimension validation</dfn>: |bindingDescriptor|.{{GPUBindGroupLayoutEntry/viewDimension}} must be {{GPUTextureViewDimension/2d}} or `undefined`.
```

Comment:
nit: I think we should say it must be 2d, and before this point IMO (can be a TODO) we should resolve all the default values. That way we don't have the default value encoded implicitly in an extra place in the spec.

Reply:
Good point, Kai. `undefined` here seems to be a little annoying and redundant because `undefined` means `2d`. However, I read the validation rules for BGL, we have `undefined` exactly in the statement for other validations (but those `undefined` are irrelevant to its default value). So removing `undefined` here might be a little bit misleading (or unclear) for web developers? I'd like to hear other reviewer's suggestion, though.

Reply:
The other references to `undefined` are in cases where the defaults don't apply, and therefore would still be `undefined` after a resolve-default-values step, so I think it would still be clearest that way.

Reply:
OK. make sense. Done!